jekyll/_includes/quick-info-list.html

```diff
@@ -0,0 +1,38 @@
+{% if page.page-type != "index" and page.page-type != "homepage" %}
+<div id="tooltip" role="tooltip" class="tooltip-popover"></div>
+<div id="tooltip-time" role="tooltip" class="tooltip-popover">
+    Last updated • Reading time
+    <div id='arrow' data-popper-arrow></div>
+</div>
+
+<div class="quick-info-list">
+    <time aria-describedby="tooltip-time" id="time-posted-on" datetime="{{  page.last_modified_at | date_to_xmlschema }}">
+    <img alt="Clock" src="{{ site.baseurl }}/assets/img/docs/duration.svg">
+        {% assign dateStart = page.last_modified_at | date: '%s' %}
+        {% assign nowTimestamp = 'now' | date: '%s' %}  
+        {% assign timestampDiff = nowTimestamp | minus: dateStart %}
+        {% assign yearInSeconds = 60 | times: 60 | times: 24 | times: 365 %}
+        {% if timestampDiff > yearInSeconds %} 1+ year ago {% else %} {{ page.last_modified_at | timeago }} {% endif %}
+    <span class="ert" title="Estimated reading time">
+        {% assign words_total = page.content | strip_html | number_of_words %}
+        {% assign words_without_code = page.content | replace: '<pre class="highlight">', '<!--' | replace: '</pre>', '-->' | strip_html | number_of_words %}
+    
+        {% assign words_code = words_total | minus: words_without_code | divided_by: 2.0 %}
+        {% assign words = words_without_code | plus: words_code | round %}
+    
+        {% assign ert = words | divided_by:250 | at_least: 1 %}
```

Comment:
I don't think this is right. Per the ticket, words should be divided by 250 and code divided by 500

Reply:
The code is divided by 2 above 
{% assign words_code = words_total | minus: words_without_code | divided_by: 2.0 %}
Then its divided by 250 which would be the same as dividing it by 500. It may be a little confusing. I can add another line in and divide it by 500 and 250 seperately then add.

Reply:
```
<span class="ert" title="Estimated reading time">
      {% assign words_total = page.content | strip_html | number_of_words %}
      {% assign words_without_code = page.content | replace: '<pre class="highlight">', '<!--' | replace: '</pre>', '-->' | strip_html | number_of_words %}
  
      {% assign words_code_read_time = words_total | minus: words_without_code | divided_by: 500 | at_least: 1%}
      {% assign words_without_code_read_time = words_without_code | divided_by: 250 | at_least: 1 %}
  
      {% assign ert = words_code_read_time | plus: words_without_code_read_time | round | at_least: 1 %}
      • {{ ert }} min read
    </span>
```